src/ebonite/client/base.py

```diff
@@ -168,16 +168,15 @@ def run_service(self, name: str, image: Image, environment: RuntimeEnvironment =
         if environment is None:
             environment = self.push_environment(RuntimeEnvironment(env_name))
 
-        params = {k: v for k, v in kwargs.items() if k in {'ports_mapping'}}
+        from ebonite.build import run_docker_img, DockerContainer
```

Comment:
and again, hardcoded docker (i know we discussed it)
i think a have a vison, how to do this 
we can add abstract method `run_service` to `RuntimeEnvironment.Params` class (and renaming it since it will be more than params container, like wrapper in `Model`) and adding implementation for each child (only docker in our case). then move code from docker helpers there (not sure about it), and in helpers write just `e = DockerEnv(); e.run_service()` or something

Reply:
Why do we need this if we already have `RunnerBase` ABC for months?

Reply:
Yes, we had it, but we did not have `RuntimeEnvironment`. But now we have (we did it for persistance), so my proposal is to merge `RunnerBase` logic inside it.

Reply:
Replied above